<launch>
  <arg name="deep_rl" default="sac_2D.py"/> 
  <arg name="ep" default="0"/> 
  <arg name="world" default="stage_1_aerial"/> 
  <arg name="root_dir" default="/home/ricardo/"/>  
  <arg name="file_dir" default="sac_stage_1/sac_env1_2d_3_layers"/>
  <arg name="graphic_int" default="false"/> 
  <arg name="testing" default="false"/> 
  <arg name="x" default="2.0"/> 
  <arg name="y" default="2.0"/> 
  <arg name="z" default="2.0"/> 
  <arg name="arr_distance" default="0.5"/> 
  <arg name="testing_eps" default="100"/> 
  <arg name="m_steps" default="500"/> 
  <arg name="e_path" default="false"/> 
  <arg name="s_2" default="false"/> 
  <arg name="max_lasers" default="false"/> 
  <arg name="min_lasers" default="true"/> 
    
  <!-- <arg name="file_dir" default="ddpg_stage_1/ddpg_env2_2d_5_layers"/>  1940 -->
  <!-- <arg name="file_dir" default="ddpg_grando/ddpg_env2_2d"/>  3660 -->
  
  <!-- <node pkg="hydrone_aerial_underwater_ddpg" type="combination_obstacle.py" name="obstacle" output="screen" /> -->
  <include file="$(find hydrone_aerial_underwater_ddpg)/launch/hydrone_aerial_deep_rl.launch" >
    <arg name="world_name" value="$(arg world)" />
    <arg name="gui" value="$(arg graphic_int)" />
    <arg name="maxi_lasers" value="$(arg max_lasers)" />
    <arg name="mini_lasers" value="$(arg min_lasers)" />
  </include>
  <node pkg="hydrone_aerial_underwater_ddpg" type="$(arg deep_rl)" name="sac_2D" output="screen">
    <param name="ep_number"  value="$(arg ep)"/>
    <param name="file_path"  value="$(arg file_dir)"/>
    <param name="test_param"  value="$(arg testing)"/>
    <param name="x_start"  value="$(arg x)"/>
    <param name="y_start"  value="$(arg y)"/>
    <param name="z_start"  value="$(arg z)"/>
    <param name="eval_path"  value="$(arg e_path)"/>
    <param name="arriving_distance"  value="$(arg arr_distance)"/>
    <param name="num_eps_test"  value="$(arg testing_eps)"/>
    <param name="max_steps"  value="$(arg m_steps)"/>
    <param name="scenario_2"  value="$(arg s_2)"/>
  </node>
  <node pkg="hydrone_aerial_underwater_ddpg" type="store_disk.py" name="store_disk" output="screen">
    <param name="file_path"  value="$(arg file_dir)"/>
    <param name="root_path"  value="$(arg root_dir)"/>
    <param name="test_param"  value="$(arg testing)"/>
    <param name="eval_path"  value="$(arg e_path)"/>
  </node>
  <node pkg="hydrone_aerial_underwater_ddpg" type="wind.py" name="wind_node" output="screen"></node>
</launch>